app/views/entry/edit.erb

```diff
@@ -0,0 +1,53 @@
+<div class="heading">
+  <h2>Edit</h2>
+</div>
+
+<div class="container">
+  <form action="/edit" method="post">
+    <input type="hidden" name="_method" value="put">
+    <input type="text" name="entry[name]" placeholder="name" value="<%= @entry.name %>" /><br>
+    <input type="text" name="entry[description]" placeholder="description" value="<%= @entry.description %>" /><br>
+    <select name="cohort">
+      <% @cohorts.each do |cohort| %>
+        <% if cohort.name == @entry.cohort.name %>
+          <option selected="selected"><%= @entry.cohort.name %></option>
+        <% else %>
+          <option value="<%= cohort.name %>"><%= cohort.name %></option>
+        <% end %>
+      <% end %>
+    </select><br>
+
+    <select name="phase">
+      <% @phases.each do |phase| %>
+        <% if phase.name == @entry.phase.name %>
+          <option selected="selected"><%= @entry.phase.name %></option>
+        <% else %>
+          <option value="<%= phase.name %>"><%= phase.name %></option>
+        <% end %>
+      <% end %>
+    </select><br>
+
+    <select name="language">
+      <% @languages.each do |language| %>
+        <% if language.name == @entry.languages.first.name %>
+          <option selected="selected"><%= language.name %></option>
+        <% else %>
+          <option value="<%= language.name %>"><%= language.name %></option>
+        <% end %>
+      <% end %>
+    </select><br>
```

Comment:
These three selects are all basically exactly the same thing with different accessors. It's probably a good case for refactoring into a partial to not have to copy/paste so much.